docs/reference_guides/model_libraries/models_extra/membrane_model/1d_membrane.rst

```diff
@@ -0,0 +1,14 @@
+One-dimensional membrane class for CO2 gas separation
+=============================================
+
+This is a one-dimensional model for gas separation in CO₂ capture applications.
+The model will be discretized in the flow direction, and it supports two flow patterns:
+counter-current flow and co-current flow. The model was customized for gas-phase separation
+in CO₂ capture with a single-layer design. If a multi-layer design is needed, multiple units
+can be connected for this application. The two sides of the membrane are called the feed side
+and sweep side. The sweep stream inlet is optional. The driving force across the membrane is the
+partial pressure difference in this gas separation application. Additionally, the energy balance
+assumes that temperature remains constant on each side of the membrane.
```

Comment:
A few comments here:

1. Reference guides should be entirely autogenerated using Sphinx. Any thing that is hand written really should be an explanation (they go else where in the docs structure).
2. It would be good to have a short code example here, or a discussion of what the inputs/degrees of freedom are. This will help users understand what information they should be supplying.

Reply:
Thanks and more info about model inputs/degrees of freedom are added.